recipe/meta.yaml

```
@@ -60,10 +44,10 @@ test:
   requires:
     - ripgrep
   commands:
-    - bash --version | rg {{ base_version }}.{{ _patch_level }}
+    - bash --version | rg {{ version }}
```

```suggestion
    - bash --version | rg {{ version }}
    - test -f $PREFIX/bin/bash
```
It would be nice to test the binary